.github/ISSUE_TEMPLATE/request-for-change.md

```diff
@@ -0,0 +1,18 @@
+---
+name: DCR Request for change
+about: Tell us about a feature that you're planning to implement in dotcom rendering
+title: "[RFC]"
+labels: RFC, dotcom-rendering
+assignees: ''
+
+---
+
+The Dotcom team would like to help you with your proposed change. Please provide us with a brief and concise explanation for what you're looking to do we will try to make the process easier with early feedback or advice.
+
+**When should this RFC be used?**
+This is an optional process. If you own the code you're working on or feel confident about the change you're making then you should simply raise a PR as normal. We want developers and teams to feel empowered to make the choice about when to reach out.
+
+But if your change affects core platform code or is introducing a new concept or pattern, or if you're just looking for more eyes and input then this process is for you.
```

Comment:
I love a “but”, but wonder if this reads nicer?


```suggestion
However, if your change affects core platform code or is introducing a new concept or pattern, or if you're just looking for more eyes and input then this process is for you.
```

Reply:
Yep